src/components/velocity.js

```diff
@@ -0,0 +1,25 @@
+var registerComponent = require('../core/component').registerComponent;
+
+/**
+ * Velocity component.
+ *
+ * Updates object's position at each tick.
+ */
+module.exports = registerComponent('velocity', {
+  dependencies: ['position'],
```

Comment:
Yeah. getAttribute returns defined values, getComputedAttribute returns all data including mixins, defaults, etc. I wonder if that's intuitive